### PR TITLE
Debug CI 

### PR DESCRIPTION
**Pull request by callmesalmon**
### .github/workflows/build-pass.yml
Typo. In the file *this* codeblock is executed:
```sh
{insignificant} && python3 smeltile.py
```
`smeltile.py` doesn't exist. Change to `smeltfile.py`.

### CODE REVIEW

File: config/config.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 set -e
 
-PROFILE_DIR="config/profiles/"
+PROFILE_DIR="config/profiles"
```
Comment (GandelXIV) [inline config/config.sh]: I believe its better to leave trailing slashes for directory paths, not that it matters much.

File: .github/workflows/build-pass.yml [hunk 2]
```diff
@@ -22,4 +22,4 @@ jobs:
       - name: configure
         run: ./config.sh default
       - name: Build x86_64
-        run: source $HOME/.cargo/env && python3 smeltile.py iso
+        run: source $HOME/.cargo/env && python3 smeltfile.py iso
```
Comment (GandelXIV) [inline .github/workflows/build-pass.yml:25]: Good catch, was wondering why it didnt work xd

File: config/config.sh [hunk 3]
```diff
@@ -29,11 +29,13 @@ mkdir -p log/
 
 echo "Configuring..."
 
-set -x
+if [ ! $PROFILE == "$PROFILE_DIR/default" ]; then
```
Comment (GandelXIV) [inline config/config.sh]: This step breaks the build process for fresh clones. Config files are not set by default in the git tree, and thus before building it is required to run this script (with `default`) to put them in their places. The default profile is currently the only one, with room for additional in the future. 

TLDR: This line breaks this, consider removing it.

File: config/config.sh
Comment (GandelXIV) [inline config/config.sh:1]: Hi, if you want to update this code, here are some things that should be added:
- abort when an invalid config profile is supplied
- allow for switching to another config without having to call `deconfig.sh`  first.
- anything you would find useful tbh
Dont forget to test it tho ;P

**Review by GandelXIV — CHANGES_REQUESTED** [review]
Hi, thanks for you contribution. There are some changes needed before we merge this.

**Comment by callmesalmon** [comment]
Yeah I reverted the config changes because I'm (believe it or not) lazy and tired. Maybe I'll come back in a few weeks with a config pull request but idk really
